tests/test_context.py

```diff
@@ -6,3 +10,28 @@ def test_ctx_load(lt_ctx, default_raw):
         dtype="float32",
         detector_size=(128, 128),
     )
+
+
+def test_ctx_load_old(lt_ctx, default_raw):
+    with warnings.catch_warnings(record=True) as w:
+        lt_ctx.load(
+            "raw",
+            path=default_raw._path,
+            scan_size=(16, 16),
+            dtype="float32",
+            detector_size_raw=(128, 128),
+            crop_detector_to=(128, 128)
+        )
+        assert len(w) == 1
+        assert issubclass(w[0].category, DeprecationWarning)
+
+
+def test_missing_detector_size(lt_ctx, default_raw):
+    with pytest.raises(ValueError) as e:
+        lt_ctx.load(
+            "raw",
+            path=default_raw._path,
+            scan_size=(16, 16),
+            dtype="float32",
+            )
+    assert e.match("missing 1 required argument: 'detector_size'")
```

Comment:
Tests look good! They should be moved to `tests/io/test_raw.py` though. You may only be calling the `load` method of the `Context`, but the behavior you are testing comes from the raw dataset.

Reply:
I thought of adding the tests to `test_raw.py` at first. But when I search for the test which open raw file normal way, It was in `test_context.py`. So I got confused.

Reply:
Ah, I can see how that is confusing. All the tests in `test_raw` just use fixtures like `default_raw` and there was no test yet that exercised the different parameters. Looks good now!